utilities/src/main/java/io/syndesis/qe/utils/SampleDbConnectionManager.java

```diff
@@ -40,6 +40,7 @@ public static Connection getConnection(String dbType) {
                 break;
         }
 
+        Assertions.assertThat(connectionsInfoMap.containsKey(dbType)).isTrue();
```

Comment:
assertThat(connectionsInfoMap).containsKey(dbType); will be better

Reply:
np